src/python/pants/help/help_info_extracter.py

```diff
@@ -144,51 +129,32 @@ class GoalHelpInfo:
     consumed_scopes: Tuple[str, ...]  # The scopes of subsystems consumed by this goal.
 
 
+def pretty_print_type_hint(hint: Any) -> str:
```

Comment:
Moved with no modification. Note that this code breaks if you use `from __future__ import annotations` in your target types definition file - that needs to be fixed in a followup.